spark-word-count/spark-k8s-container-image/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM svangapallycs/spark-base:fLmLt0
+
+USER root
+
+COPY target/*.jar target/
+
+RUN apt-get update -y && apt-get install -y wget && wget https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/2.7.0/hadoop-aws-2.7.0.jar -P ${SPARK_HOME}/jars/ && wget https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk/1.7.4/aws-java-sdk-1.7.4.jar -P ${SPARK_HOME}/jars/ && wget https://repo1.maven.org/maven2/com/googlecode/json-simple/json-simple/1.1/json-simple-1.1.jar -P ${SPARK_HOME}/jars/
```

Comment:
break this into a few `RUN` commands

Reply:
Should I break this into installing & downloading parts?